install-panel.sh

```diff
@@ -227,8 +224,7 @@ function configure {
   php artisan p:environment:mail
 
   # configures database
-  print_warning "You must type 'yes' or else the installer will fail! The default response 'no' will not properly initialize the database!"
-  php artisan migrate --seed
+  php artisan migrate --seed --force
```

Comment:
@danielnaaman Is it this step you are referring to? This is where I added the `--force` flag which will skip the `yes` prompt.

Reply:
> @danielnaaman Is it this step you are referring to? This is where I added the `--force` flag which will skip the `yes` prompt.

My bad haha